imap_processing/lo/l0/lo_science.py

```diff
@@ -470,14 +473,27 @@ def organize_spin_data(dataset: xr.Dataset) -> xr.Dataset:
         "period_source_spin",
     ]
 
+    # Set epoch to the acq_start time
+    acq_start = dataset.acq_start_sec.values + (0.1 * dataset.acq_start_subsec.values)
+    epoch = met_to_ttj2000ns(acq_start)
```

Comment:
I've noticed that we do a lot of `.values` / `.data` on the project, does the `met_to_ttj2000ns()` function not take array-like and it has to be numpy specifically? In general, we should be able to use `dataset["acq_start_sec"]` and that will act like an array 🤔

Reply:
Yes, that does work. Not sure why I thought I had to do `.values`

Reply:
I think we are doing this all over the place, so it is a pretty common pattern.